sdk/Sdk.Generators/Properties/launchSettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "profiles": {
-    "SourceGenDebug": {
+    "Source Generator": {
```

Comment:
Could having a space in the name lead to issues? `SourceGenerator` feels safer to me for some reason

Reply:
There shouldn't be an issue. This is mainly a display string for Visual Studio launch/debug dropdown. But I am also not against removing the space if you feel strongly for that.